src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,25 @@
+import { screen } from "@testing-library/react";
+
+import { default as Home } from ".";
+
+describe("Home page", () => {
+  test("it renders a message on error", async () => {
+    const initialState = {
+      username: "tester",
```

Comment:
TEST FAILS AHAHAHAHAHAHAHAHHA, dunno why though tbf looks like it works, did you forget to change the test error message after changing the website?

Reply:
hmm... I'm not sure actually I'll have a look.
Everything passed the last time we ran them